seguro/commands/notebook_executor/main.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: 2023 Steffen Vogel, OPAL-RT Germany GmbH
+# SPDX-License-Identifier: Apache-2.0
+
+import os.path
+import sys
+
+import nbformat
+from nbconvert import PDFExporter
+from nbconvert.preprocessors import ExecutePreprocessor
+
+from seguro.common import store, job
+
+
+def main() -> int:
+    client = store.Client()
+
+    if job.info.triggered_by.event != "created":
+        return -1
+
+    nb_obj = job.info.notebook
+
+    # Fetch Notebook from store
+    nb_contents = client.get_file_contents(nb_obj).read()
+
+    # Read Notebook
+    nb = nbformat.reads(nb_contents, as_version=4)
+
+    # Execute Notebook
+    ep = ExecutePreprocessor(timeout=600, kernel_name="python3")
+    ep.preprocess(nb)
+
+    # Export Notebook as PDF file
+    pexp = PDFExporter()
```

Comment:
I think this should be an optional feature as otherwise the object store will get cluttered with PDF files for each notebook execution.

Reply:
Right know, we simply name the PDF file according to the iPyNb file. So repeated executions would simply overwrite the old PDF.

I think we should also always export the executed Notebook, as otherwise we have no insight to what the Notebook results are?

We are currently only pushing the exported PDF to the object store, not the evaluated notebook file.

But that makes me think: Should we also upload the new Notebook back to the store? Or just the exported PDF?